x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/bucket_span/description.tsx

```diff
@@ -25,11 +25,11 @@ export const Description: FC<Props> = memo(({ children, validation }) => {
       description={
         <FormattedMessage
           id="xpack.ml.newJob.wizard.pickFieldsStep.bucketSpan.description"
-          defaultMessage="Set the interval for time series analysis, typically between 15m to 1h."
+          defaultMessage="The interval for time series analysis, typically between 15m to 1h."
```

Comment:
Reading this sentence again has me wondering whether it should be `typically between 15m and 1h`, or `typically 15m to 1h`. Any thoughts @szabosteve @lcawl ?